### PR TITLE
Download data

### PR DESCRIPTION
**Pull request by juliamorka**
Added script that downloads the IMGW data. Created template data directory and constants definition. Code needs refactoring, moving modules into right directories and unit tests in later development stages.

### CODE REVIEW

**Comment by juliamorka** [comment]
TODO: Add conda .yml environment

**Comment by juliamorka** [comment]
@jstaszel-agh the code is ready for a review

**Comment by juliamorka** [comment]
@jstaszel-agh, could you review the code? It's been a week since I created the merge request and it's blocking me from further development. Thank you.

File: src/imgw_pipeline.py
Comment (jstaszel-agh) [inline src/imgw_pipeline.py:1]: I would suggest to keep this pipeline outside of the src package, where only source code should be kept.
You can leave the pipeline in the main directory of the repository or create a new called e.g. "scipts".

File: env.yml
Comment (jstaszel-agh) [inline env.yml:1]: I recommend to add conda-lock and in this file just keep packages and versions you are consciously using.

File: src/imgw_pipeline.py
Comment (jstaszel-agh) [inline src/imgw_pipeline.py:1]: That would be good to add some information in the console during running the scripts, e.g. by using print or logging.

**Review by jstaszel-agh — APPROVED** [review]
I am really sorry for the delay! The repo name was misleading for me, I assumed it is a repo for a different subject where it is not that important to accept PRs.
You can also have an additional branch, like named "dev" where you keep your final code and then create PRs for me from dev to prod. In this way I won't block your progress.

File: src/imgw_pipeline.py
Comment (juliamorka) [reply src/imgw_pipeline.py:1]: That's a great idea, I will add this as an issue to the repo and work on it if I have some time left before the assignment due date. Thank you!

**Comment by juliamorka** [comment]
All of the issues have been adressed, thank you for the tips!

**Pull request merged**